plugin/pkg/scheduler/scheduler.go

```diff
@@ -46,10 +50,11 @@ type Scheduler struct {
 type Config struct {
 	// It is expected that changes made via SchedulerCache will be observed
 	// by NodeLister and Algorithm.
-	SchedulerCache schedulercache.Cache
-	NodeLister     algorithm.NodeLister
-	Algorithm      algorithm.ScheduleAlgorithm
-	Binder         Binder
+	SchedulerCache      schedulercache.Cache
+	NodeLister          algorithm.NodeLister
+	Algorithm           algorithm.ScheduleAlgorithm
+	Binder              Binder
+	PodConditionUpdater PodConditionUpdater
```

Comment:
PodConditionUpdate is currently used only to update pod condition in case of scheduling error. That said, if we decide to use it also in case of successful scheduling, binding and changing pod condition will not be atomic. We should probably add an explicit comment about it.